html/index.html

```diff
@@ -205,111 +205,144 @@ <h2 class="my-0 fw-semibold">Nautobot Core</h2>
                 <div class="col">
                     <div class="card mb-4 rounded-3 shadow-sm h-100">
                         <div class="card-header py-3">
-                            <h2 class="my-0 fw-semibold">Nautobot Apps</h2>
+                            <h2 class="my-0 fw-semibold">Open Source Apps</h2>
                         </div>
                         <div class="card-body">
-                            <p class="card-text">The documentation portal for all Nautobot Applications. Learn about all
-                                of the Apps in the ecosystem with guides for Usage, Administration, and
-                                Development of each App.</p>
+                            <p class="card-text">This section lists all of the open source Nautobot Apps developed by
+                                Network To Code. Learn about these Apps with guides for Usage, Administration, and
+                                Development.</p>
                         </div>
                         <ul class="list-group list-group-flush border-bottom-0">
                             <li class="list-group-item"><a
-                                    href="https://docs.nautobot.com/projects/core/en/stable/apps/"
-                                    class="w-100 btn btn-lg btn-warning" target="_blank">To the App Docs!</a>
+                                    href="https://docs.nautobot.com/projects/core/en/stable/development/apps/"
```

Comment:
In my opinion the previous link was more appropriate because it sent you to a list of apps. This link sends you to the app development guide.

Reply:
Unfortunately the previous link is to a section that's not really kept up to date and a bit redundant with other websites we have (the various app marketplaces, the ntc website etc). I switched over to the app dev guide as the next best thing for now, which I know isn't ideal either.

I'll get back to this once we align our websites better and perhaps link to the ntc.com page.